templates/default/luigi-server.init.erb

```diff
@@ -0,0 +1,158 @@
+#! /bin/sh
+### BEGIN INIT INFO
+#
+# Created by Chef
+#
+# Provides:          skeleton
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Example initscript
+# Description:       This file should be used to construct scripts to be
+#                    placed in /etc/init.d.
+### END INIT INFO
+
+# Do NOT "set -e"
+
+# PATH should only include /usr/* if it runs after the mountnfs.sh script
+PATH=/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin
+DESC="Luigi central planner server"
+NAME=luigid
+DAEMON=/usr/local/bin/$NAME
+PIDFILE=<%= @luigid_pidfile %>
+LOGDIR=<%= @luigid_logdir %>
+DAEMON_ARGS="--background --pidfile $PIDFILE --logdir $LOGDIR"
+SCRIPTNAME=/etc/init.d/$NAME
+
+# Exit if the package is not installed
+[ -x "$DAEMON" ] || exit 0
+
+# Read configuration variable file if it is present
+[ -r /etc/default/$NAME ] && . /etc/default/$NAME
```

Comment:
Does it make sense to create `luigi-env.sh.erb` in `/etc/default/luigid`?
If so, we could define attributes like `default['luigid']['env']['VALUE']` to be include in the template file.